tests/test_mesh.py

```diff
@@ -2,12 +2,149 @@
 import trimesh
 import numpy as np
 import os
+from huggingface_hub import snapshot_download
 
 import genesis as gs
 import genesis.utils.mesh as mu
 import genesis.utils.gltf as gltf_utils
+import genesis.utils.usda as usda_utils
+from .utils import assert_allclose, assert_array_equal
 
+VERTICES_TOL = 1e-05  # Transformation loses a little precision in vertices
+NORMALS_TOL = 5e-02  # Conversion from .usd to .glb loses a little precision in normals
```

Comment:
Actually `5e-2` seems to be very large. For a distance it does not sound acceptable.

Reply:
1e-02 is the best we can do. The normal is changed a bit when @ACMLCZH convert usd to glb using blender. Maybe we have to go with this.